docs/extend/packaging-zos-extensions.md

```diff
@@ -31,6 +32,10 @@ A typical component package, for example, `jobs-api-package-1.0.4.zip`, consists
 - `manifest.yaml`
 
    Refers to the Zowe component manifest file. You can find detailed definition of manifest in [Server Component Manifest File Reference](../appendix/server-component-manifest.md).
+   
+- `schema.json`
+
+   Refers to the [json schema](https://json-schema.org/) file that details the parameters that are valid for the component's configuration within Zowe server configuration files. See documentation on [server component schema files](server-schemas.md) for more information.
```

Comment:
Is it worthy to clarify that`schema.json` file name is customizable by defining in `manifest.yaml` `schemas.configs` section.

Reply:
yes